src/plugin/serve.ts

```diff
@@ -164,6 +164,12 @@ export const createServeCommand = (plugin: Plugin) => {
         choices: TELEMETRY_LEVEL_CHOICES,
         default: 'all',
       },
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      license: {
```

Comment:
This adds the flag to both the package and serve commands, don't we need it only for serve? If so it should be added in https://github.com/cloudquery/plugin-sdk-javascript/blob/558a8287246e86e686616c8bb3ba095b7cf41edf/src/plugin/serve.ts#L43

Or we can remove the `strict` part so new flags are ignored

Reply:
Yes it should be only for serve. Removing strict makes it so? I'm not sure if making it ignore unknown flags is wise, at least we have a TODO list in terms of unimplemented flags now.

Reply:
OK let's just add it and be done with it here https://github.com/cloudquery/plugin-sdk-javascript/blob/558a8287246e86e686616c8bb3ba095b7cf41edf/src/plugin/serve.ts#L43

Reply:
added in https://github.com/cloudquery/plugin-sdk-javascript/pull/108/commits/46e0cb98493d204c18b3ca91ce0267eaca57f40c

Reply:
Pushed a commit to do this